src/decode/stream.rs

```diff
@@ -0,0 +1,510 @@
+use crate::decode::lzbuffer::{LZBuffer, LZCircularBuffer};
```

Comment:
Note: I didn't look in detail at this file (yet). But it is restricted to the `stream` feature.